client/components/fields/editor/AssociatedEventItem.tsx

```diff
@@ -107,11 +117,22 @@ export class AssociatedEventItem extends React.PureComponent<IProps> {
                                         ['_endTime', '_startTime', '_created', '_etag', '_links', '_updated']
                                     );
 
-                                    return eventsApi.events.create(itemClean).then(([createdEvent]) => {
-                                        this.update(createdEvent);
+                                    // Sometimes UI is clicked too fast, and since we don't have a perfect flow
```

Comment:
You mean plus button? Can you disable the button after the first click until promise resolves? It's not clean to handle it here